gensim/models/word2vec.py

```diff
@@ -452,6 +453,9 @@ def __init__(
         self.total_train_time = 0
         self.sorted_vocab = sorted_vocab
         self.batch_words = batch_words
+        
+        self.load = methodize(load, self)
```

Comment:
This was done on purpose. Please see [here](http://stackoverflow.com/a/861109) for reference.

`MethodType` binds the method to the calling instance. What I'm doing here is something like overloading `load` based on whether it is called on instance or class.

So the argument `load` to `methodize` is the name of the function that is to be bound.